src/components/dashboard-lists.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
+export class Lists extends Component {
```

Comment:
Generally, if your file is `dashboard-lists` your default export should be `DashboardLists`